app/src/utils/enums/housing.ts

```diff
@@ -80,11 +80,14 @@ export enum PermitStatus {
   PENDING = 'Pending decision'
 }
 
+export enum ProjectApplicant {
+  BUSINESS = 'Business',
+  INDIVIDUAL = 'Individual'
+}
+
 export enum ProjectRelationship {
-  OWNER = 'Owner',
-  EMPLOYEE = 'Employee',
-  AGENT = 'Agent',
-  CONSULTANT = 'Consultant',
+  OWNER = 'Property owner',
+  CONSULTANT = 'Project consultant',
```

Comment:
Data Standards: I'm not too certain on the linkages between these actors, but I would recommend double checking to see if this enum should be changing to these descriptors with adjectives attached, as these terms may be more specific than is necessary.